src/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/MathHelpers.cs

```diff
@@ -215,6 +218,110 @@ public static long Flt2LngOvf(float val)
             return ThrowIntOvf();
         }
 
+#if ARM
+        private const string RuntimeLibrary = "[MRT]";
+
+        [RuntimeImport(RuntimeLibrary, "RhpIDiv")]
+        private static extern Int32 RhpIDiv(Int32 i, Int32 j);
+
+        [RuntimeExport("IDiv")]
```

Comment:
Do these need to be `RuntimeExports`? What refers to them by the symbolic name?

Reply:
@MichalStrehovsky I don't know the answer about ```RuntimeExports```. Which way can I call  C++ ```RhpIDiv``` from CS ```IDiv```?

Reply:
> Which way can I call C++ RhpIDiv from CS IDiv?

`RuntimeExport` doesn't affect calls from managed code to unmanaged code. `RuntimeExport` will expose the managed method under the specified symbolic name so that a different component can refer to it under that name. I don't think it's needed here.

Reply:
@MichalStrehovsky  thanks!